### ISSUE TITLE
Accessory does not appear in Home

### ISSUE THREAD

**Issue by a-dows** [grzegorz914/homebridge-denon-tv#222]
I removed my AVR-S760H from Homekit due to occasional "not responding" issues. I have not been able to add it back, even with uninstalling and reinstalling the plugin. If I create a child bridge, the child bridge can be added but not the accessory. Seeing the following error in the log:

`Device: 192.168.13.46 AVR, prepare accessory error: TypeError: Cannot read properties of undefined (reading 'SAT/CBL')`

I am also experiencing the issue here: https://github.com/grzegorz914/homebridge-denon-tv/issues/190, so I've got a manual config:

```
            "platform": "DenonTv",
            "devices": [
                {
                    "name": "AVR",
                    "host": "192.168.13.46",
                    "port": 8080,
                    "zoneControl": 0,
                    "getInputsFromDevice": true
                }
```

This has been occurring since 4.x.

**Comment by grzegorz914** [comment]
Go to homebridge/denontv/ folder and remove inputs, inputsNames, inputsTargetVisibility then restart plugin.

**Comment by grzegorz914** [comment]
#223

**Comment by kleingunni** [comment]
have same problem. delete inputs, input names, inputs Target Visibility without success

**Comment by grzegorz914** [comment]
Debug log please

**Comment by kleingunni** [comment]
Device: 192.168.168.100 AVR, prepare accessory error: TypeError: Cannot read properties of undefined (reading 'SAT/CBL')

**Comment by grzegorz914** [comment]
Go to plugin config Advanced Settings>>System>>Debug enable it and get the whole log.

**Comment by kleingunni** [comment]
Bug was fixed with version 4.5.6. thanks for the great support

**Comment by rockychf** [comment]
> I removed my AVR-S760H from Homekit due to occasional "not responding" issues. I have not been able to add it back, even with uninstalling and reinstalling the plugin. If I create a child bridge, the child bridge can be added but not the accessory. Seeing the following error in the log:
> 
> `Device: 192.168.13.46 AVR, prepare accessory error: TypeError: Cannot read properties of undefined (reading 'SAT/CBL')`
> 
> I am also experiencing the issue here: #190, so I've got a manual config:
> 
> ```
>             "platform": "DenonTv",
>             "devices": [
>                 {
>                     "name": "AVR",
>                     "host": "192.168.13.46",
>                     "port": 8080,
>                     "zoneControl": 0,
>                     "getInputsFromDevice": true
>                 }
> ```
> 
> This has been occurring since 4.x.

I have totally same error with it for my AVC-A110, and I tried to updated to latest version to 4.5.9 and still had the same result.
